catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -98,9 +98,10 @@
 	public static final String TEXT_TO_SPEECH_TMP_PATH = TMP_PATH + "/textToSpeech";
 
 	// Web:
-	private static final String MAIN_URL_PRODUCTION = "https://share.catrob.at";
-	private static final String MAIN_URL_WEBTEST = "https://web-test.catrob.at";
-	public static final String MAIN_URL_HTTPS = BuildConfig.USE_WEB_TEST ? MAIN_URL_WEBTEST : MAIN_URL_PRODUCTION;
+	//private static final String MAIN_URL_PRODUCTION = "https://share.catrob.at";
```

Comment:
delete code if it isn't needed any more only leave comments if really necessary / helping to explain something